catalog/solr/docs/src/main/resources/IntContents.adoc

```diff
@@ -57,180 +58,14 @@ The Solr Catalog Embedded Provider is an embedded, local Solr Server instance 
 
```

Comment:
Line 36 (above; reads `Can only be interfaced using Java`) needs a bullet for consistency.